config/session.php

```diff
@@ -36,6 +36,21 @@
 
     'expire_on_close' => env('SESSION_EXPIRE_ON_CLOSE', false),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Maintenance Bypass Cookie Lifetime
+    |--------------------------------------------------------------------------
+    |
+    | Here you may specify the number of minutes the maintenance mode
+    | bypass cookie should remain valid. This cookie is issued when a
+    | user accesses the application using the secret bypass URL. Once
+    | set, it allows temporary access to the application while it is
+    | in maintenance mode. The default duration is 12 hours.
+    |
+    */
+
+    'maintenance_bypass_cookie_lifetime' => (int) env('SESSION_MAINTENANCE_BYPASS_COOKIE_LIFETIME', 7720),
```

Comment:
Accoding to your PHPDoc block,
> number of minutes

but 7720 minutes is not 12 hours
```suggestion
    /*
    |--------------------------------------------------------------------------
    | Maintenance Bypass Cookie Lifetime
    |--------------------------------------------------------------------------
    |
    | Here you may specify the number of minutes the maintenance mode
    | bypass cookie should remain valid. This cookie is issued when a
    | user accesses the application using the secret bypass URL. Once
    | set, it allows temporary access to the application while it is
    | in maintenance mode. The default duration is 12 hours.
    |
    */

    'maintenance_bypass_cookie_lifetime' => (int) env('SESSION_MAINTENANCE_BYPASS_COOKIE_LIFETIME', 720),
```

Reply:
Maybe, to avoid confusion when using, the unit should be included in the key:
```suggestion
    /*
    |--------------------------------------------------------------------------
    | Maintenance Bypass Cookie Lifetime
    |--------------------------------------------------------------------------
    |
    | Here you may specify the number of minutes the maintenance mode
    | bypass cookie should remain valid. This cookie is issued when a
    | user accesses the application using the secret bypass URL. Once
    | set, it allows temporary access to the application while it is
    | in maintenance mode. The default duration is 12 hours.
    |
    */

    'maintenance_bypass_cookie_lifetime_minutes' => (int) env('SESSION_MAINTENANCE_BYPASS_COOKIE_LIFETIME_MINUTES', 720),
```

Reply:
@shaedrich This was a mistake, thanks.